indico/modules/rb_new/controllers/backend/common.py

```diff
@@ -16,16 +16,24 @@
 
 from __future__ import unicode_literals
 
+from marshmallow import fields as mmfields
 from marshmallow_enum import EnumField
 from webargs import fields
 
 from indico.legacy.common.cache import GenericCache
 from indico.modules.rb.models.reservations import RepeatFrequency
+from indico.modules.users import User
 
 
 _cache = GenericCache('Rooms')
 
 
+class UserField(mmfields.Field):
```

Comment:
I wonder if it wouldn't be better to create a generic field here...

Maybe a single-object version (`ModelObject`?) of [`ModelList`](https://github.com/indico/indico/blob/master/indico/util/marshmallow.py#L54). The `__init__` logic would be the same, just a slightly different `_deserialize` implementation